contrib/palantirjavaformat/src/mill/contrib/palantirjavaformat/PalantirJavaFormatModule.scala

```diff
@@ -0,0 +1,187 @@
+package mill
+package contrib.palantirjavaformat
```

Comment:
I edited a comment earlier to say this, but let's standardize the naming convention of `palantirformat`/`PalantirFormat`. I know it doesn't quite match the `palantir-java-format` project name, but using the full name over and over is getting verbose enough I think we should shorten it here just for ease of use

Reply:
@lihaoyi 
How about `palantirjavafmt`? Get a lot more clarity with the addition of a single character.

Reply:
Let's go with `palantirformat`, googling `palantir format` brings up only this one project, so there shouldn't be any ambiguity for now

Reply:
One consideration is that I may want to move this directly into `javalib`, just like how `ScalaFmtModule` is in `scalalib` and `KLintModule` is in `kotlinlib`. That way it'll be `mill.javalib.palantirformat` which should be unambiguous enough

Reply:
Where would the documentation go in this case? `docs/modules/ROOT/pages/Java_Module_Config.adoc`?